static/js/markdown.js

```diff
@@ -163,8 +164,9 @@ function handleEmoji(emoji_name) {
                ' title="' + title + '">';
     } else if (emoji_codes.name_to_codepoint.hasOwnProperty(emoji_name)) {
         var codepoint = emoji_codes.name_to_codepoint[emoji_name];
-        return '<span class="emoji emoji-' + codepoint + '"' +
-               ' title="' + title + '">' + alt_text +
+        return '<span aria-label="' + title + '"' +
+               ' class="emoji emoji-' + codepoint + '"' +
+               ' role="img" title="' + title + '">' + alt_text +
                '</span>';
```

Comment:
Might be worth extracting this as a function `make_emoji_span(codepoint, title, alt_text)` for readability, since we now call it in two places.

Reply:
Agree, done! :)